src/ansys/mapdl/core/cli/hpc.py

```diff
@@ -72,11 +73,34 @@
     help="""Python version to use to create the virtual environment and
 run the Python file. Python3 is used by default in the cluster.""",
 )
+@click.option(
+    "--inputs",
+    default=None,
+    type=str,
+    help="""
+Input arguments for the simulation. You can specify several arguments by
+joining them with commas. Thus, strings defined in this way cannot contain
+commas. Only integers, floats and strings are allowed.
+PyMAPDL converts these inputs to integer or float values when possible.
+Otherwise, they remain as strings. You can change these arguments on the
+HPS website. For example, ``--inputs="force=123,value='mystring'"``.
+    """,
+)
+@click.option(
+    "--outputs",
+    default=None,
+    type=str,
+    help="""Output parameters. You can specify several arguments
+by joining them with commas.
+For example, ``--outputs="displacements,nodes"``.""",
+)
 @click.option(
     "--output_files",
     default=None,
     type=str,
-    help="""Output files to monitor.""",
+    help="""Output files to monitor. Because you use commas to separate
+the file names, the names cannot contain commas. For example, 
```

Comment:
```suggestion
    help="""Output files to monitor. Because you use commas to separate
the file names, the names cannot contain commas. For example,
```